tsdat/io/converters/string_to_datetime.py

```diff
@@ -49,6 +49,8 @@ class StringToDatetime(DataConverter):
     """Any parameters set here will be passed to `pd.to_datetime` as keyword
     arguments."""
 
+    # TODO: Though not strictly a problem, it's generally bad form to name a method parameter the
+    #  same as a class variable.
     @validator("format")
     @classmethod
     def warn_if_no_format_set(cls, format: Optional[str]) -> Optional[str]:
```

Comment:
Just clarifying – this function is called automatically by `pydantic` to validate the class arguments when an instance of `StringToDatetime` is created. In this case it's been set up to validate only the `format` value of the class (`@validator("format")` controls that part). Would you still recommend using a different variable name in the method signature?

Reply:
Either that or renaming the class variable. I wouldn't call this a must-do either, it's more just to prevent any scoping issues that *may* arise in more niche cases. I don't see it as a problem so much as just not falling in line with established Python norms. We can safely ignore it, but I did want to at least raise this one.